### PR TITLE
Convert to toml packaging

### PR DESCRIPTION
**Pull request by hareskoug**
Replace the old `setup.py` package definition with a `.toml` definition.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "visier-connector-python"
+authors = [
+    {name = "Visier Research & Development", email = "info@visier.com"},
+]
+description = """Visier People Data connector"""
```
Comment (domMayhew) [inline pyproject.toml:10]: This is different from the previous description (Visier People Data connector through the Visier SQL-like API). Is this intentional?

Comment (domMayhew) [inline pyproject.toml:10]: There's no long description in the new file. Intentional?

File: setup.py [hunk 2]
```diff
@@ -1,57 +0,0 @@
...
-"""
-
-from setuptools import setup, find_packages  # noqa: H301
-import visier
-
-
-NAME = "visier-connector"
-VERSION = visier.__version__
-
-# To install the library, run the following
```
Comment (domMayhew) [inline setup.py:25]: This comment didn't make it in to the new file. Is this or any other comments required?

File: pyproject.toml (hunk 1 above)
Comment (hareskoug) [reply pyproject.toml:10]: Yes, the connector is more generic than that description would suggest

Comment (hareskoug) [reply pyproject.toml:10]: Yes, the long description is replaced by the reference to the `README.md`

File: setup.py (hunk 2 above)
Comment (hareskoug) [reply setup.py:25]: Correct. I don't think it would add anything meaningful

**Review by domMayhew — APPROVED** [review]

**Pull request merged**